tasks/db_deploy/install/create_db.py

```diff
@@ -5,14 +5,18 @@
 """
 from typing import Dict, List
 
-from orca_shared.database.shared_db import get_admin_connection, logger
+from aws_lambda_powertools import Logger
+from orca_shared.database.shared_db import get_admin_connection
 from orca_shared.reconciliation.shared_reconciliation import (
     get_partition_name_from_bucket_name,
 )
 from sqlalchemy.future import Connection
 
 import install.orca_sql as sql
 
+# Set AWS powertools logger
+LOGGER = Logger()
```

Comment:
Recommend reverting here, and adding to card to standardize logger propagation.

Reply:
removed